src/utils/validator.ts

```diff
@@ -247,3 +247,19 @@ export function isTopic(topic: any): boolean {
   const VALID_TOPIC_REGEX = /^(\/topics\/)?(private\/)?[a-zA-Z0-9-_.~%]+$/;
   return VALID_TOPIC_REGEX.test(topic);
 }
+
+/**
+ * Validates that a value is a number and falls within the given range.
+ *
+ * @param {any} value The value to validate.
+ * @param {min} lower bound of the range to validate.
+ * @param {max} upper bound of the range to validate.
+ * @param {inclusive} indicates whether to include or exclude lower and upper bounds in the range.
+ * @return {boolean} Whether the value is a number or not and weather the value is within the range.
+ */
+export function isNumberInRange(value: any, min: number, max: number, inclusive: boolean = false) {
```

Comment:
Lets also move under `messaging/` for now.

Reply:
Since we will be removing the `Color` class we will no longer need this validation... I will remove this, too.